code/modules/ventcrawl/ventcrawl.dm

```diff
@@ -3,6 +3,15 @@ var/global/list/ventcrawl_machinery = list(
 	/obj/machinery/atmospherics/unary/vent_pump
 	)
 
+/mob/living/carbon/human/AltClickOn(var/atom/A)
+	if(is_type_in_list(A,ventcrawl_machinery) && w_uniform && istype(w_uniform,/obj/item/clothing/under/contortionist))
```

Comment:
grumble: I _hate_ single-type typecheck carveouts, though with the garbage legacy architecture there's not much we can do to work around this without making this a whole huge PR. Something something ECS something.

Reply:
Also, I would flip the order of checks here, is_type_in_list at the end. 

The other checks are much cheaper, and if any fails the rest won't run (short-circuiting).

Reply:
Quickly popping in with my usual `istype` _already_ checks for nulls. You don't need the added `&& w_uniform`